sycl/test-e2e/Basic/launch_queries/max_num_work_groups.cpp

```diff
@@ -161,9 +161,9 @@ int test_max_num_work_groups(sycl::queue &q, const sycl::device &dev) {
   // It cannot be possible to launch a kernel successfully with a configuration
   // that exceeds the available resources as in the above defined workGroupSize.
   // workGroupSize is larger than maxWorkGroupSize, hence maxWGs must equal 0.
-  // Note: Level-Zero currently always returns a non-zero value. While other
-  // backends (i.e., OpenCL, HIP) always return 1 in their implementations.
-  if (dev.get_backend() == sycl::backend::ext_oneapi_cuda) {
+  // Note: Level-Zero currently always returns a non-zero value.
+  // TODO: Remove the backend condition once the Level-Zero API issue is fixed.
+  if (dev.get_backend() != sycl::backend::ext_oneapi_level_zero) {
```

Comment:
This is a small thing, but can we improve the assert message?  It seems that the message we want some other developer in the future to understand is that the query SHOULD have failed (but didn't).